sos/report/plugins/flatpak.py

```diff
@@ -19,6 +19,7 @@ class Flatpak(Plugin, IndependentPlugin):
     packages = ("flatpak",)
 
     def setup(self):
+        env = {"GVFS_REMOTE_VOLUME_MONITOR_IGNORE": 1}
```

Comment:
This looked ok at first blush, but the smoke test is failing with `TypeError: expected str, bytes or os.PathLike object, not int`.

I'm figuring we'll need to quote that integer here.

Reply:
Indeed, `Popen` requires `env` as a `dict` from `str` to `str` only. One still learns  new things.. :)

Fixed now.